xml/System.Web.UI.WebControls/ListBox.xml

```diff
@@ -594,7 +594,7 @@
       </ReturnValue>
       <Docs>
         <summary>Gets or sets the selection mode of the <see cref="T:System.Web.UI.WebControls.ListBox" /> control.</summary>
-        <value>One of the <see cref="T:System.Web.UI.WebControls.ListSelectionMode" /> values. The default value is <see langword="Single" />.</value>
+        <value>One of the <see cref="T:System.Web.UI.WebControls.ListSelectionMode" /> values. The default value is <see cref="T:System.Single" />.</value>
```

Comment:
This one seems to be referring to an enum value. It should also not be using langword, but the API in the cref needs to be different.